psalm.xml

```diff
@@ -39,5 +39,24 @@
                 <file name="tests/Doctrine/Tests/Persistence/Mapping/SymfonyFileLocatorTest.php"/>
             </errorLevel>
         </NullArgument>
+        <PossiblyNullReference>
+            <errorLevel type="suppress">
+                <!-- ReflectionProperty::getType() can return null, but there is ReflectionProperty::hasType()
+                check before -->
+                <file name="lib/Doctrine/Persistence/Reflection/TypedNoDefaultReflectionProperty.php"/>
+            </errorLevel>
+        </PossiblyNullReference>
```

Comment:
For things like this that can be fixed in next psalm releases, can we maybe use annotations so we can enable [findUnusedPsalmSuppress](https://psalm.dev/docs/running_psalm/configuration/#) and if this is fixed, psalm would warn us.

Reply:
Yes please!

Reply:
Hmm I added this in the first place because locally I used `vendor/bin/psalm --no-cache --php-version=7.4` and running that command it throws the `PossiblyNullReference`.

But since we are not specifying the php version here, I think it's using the lowest one from `composer.json`, `7.1` and using that one there is no error.